homeassistant/components/hue/sensor.py

```diff
@@ -66,3 +67,25 @@ def state(self):
             return None
 
         return self.sensor.temperature / 100
+
+class HueSwitch(GenericHueGaugeSensorEntity):
+    """The dimmer sensor entity for a Hue sensor device."""
+
+    device_class = DEVICE_CLASS_SWITCH
+
+    @property
+    def state(self):
+        BUTTONS = {
+                    34: "1_click", 16: "2_click", 17: "3_click", 18: "4_click",
+                    1000: "1_click", 2000: "2_click", 3000: "3_click", 4000: "4_click",
+                    1001: "1_hold", 2001: "2_hold", 3001: "3_hold", 4001: "4_hold",
+                    1002: "1_click_up", 2002: "2_click_up", 3002: "3_click_up", 4002: "4_click_up",
+                    1003: "1_hold_up", 2003: "2_hold_up", 3003: "3_hold_up", 4003: "4_hold_up",
+        }
```

Comment:
These should be more human-readable values if they are going to be presented in the UI - lose the underscores and consider rephrasing them:
* "button 1 clicked"
* "button 1 held"
* "button 1 hold released"

Reply:
This should be removed and make the switch only signal through event. Use documentation to give examples on how the button events behave